### PR TITLE
skinnywms

### PR DESCRIPTION
**Pull request by sylvielamythepaut**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/skinnywms/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - skinnywms
+
+about:
+  home: http://github.com/ecmwf/skinnywms
+  license: Apache
```
Comment (marcelotrevisani) [inline recipes/skinnywms/meta.yaml]: Could you please change it to:
```suggestion
  license: Apache-2.0
  license_family: Apache
```

File: recipes/skinnywms/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  # The remaining entries in this section are optional, but recommended.
+  description: 'Visualising meteorological and climate data'
+  doc_url: http://github.com/ecmwf/skinnywms
+  dev_url: https://github.com/ecmwf/skinnywms
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
+    # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
```
Comment (marcelotrevisani) [inline recipes/skinnywms/meta.yaml]: Could you please remove those comments?

File: recipes/skinnywms/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  skip: True  # [py<35]
```
Comment (marcelotrevisani) [inline recipes/skinnywms/meta.yaml]: Could this recipe be ``noarch``?
```suggestion
  noarch: python
```
and if you want to create packages just for python 3.6+ , you can add some selector in the ``host`` and ``run`` requirements

File: recipes/skinnywms/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:  
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/skinnywms/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/skinnywms/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    - pip
+    - magics
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/skinnywms/meta.yaml]:
```suggestion
    - python >3.5
```

File: recipes/skinnywms/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - skinnywms
+
+about:
+  home: http://github.com/ecmwf/skinnywms
+  license: Apache
+  license_file: LICENSE.txt
```
Comment (marcelotrevisani) [inline recipes/skinnywms/meta.yaml]: I opened a PR on this repository to add the license file to the project and also to include it to the manifest, after that you not be necessary to include the license manually with this recipe

Comment (marcelotrevisani) [reply recipes/skinnywms/meta.yaml]: https://github.com/sylvielamythepaut/skinnywms/pull/1

Comment (sylvielamythepaut) [reply recipes/skinnywms/meta.yaml]: I had put the LICENSE.txt in the recipe/skinnywms .. Is it not the right place ?

File: recipes/skinnywms/meta.yaml (hunk 2 above)
Comment (sylvielamythepaut) [reply recipes/skinnywms/meta.yaml]: Thanks, I do that immediately

File: recipes/skinnywms/meta.yaml (hunk 6 above)
Comment (marcelotrevisani) [reply recipes/skinnywms/meta.yaml]: Yes, it is! :)
But we do that when the project does not ship the license. We try to avoid it as much as possible.
If there is a possibility to ship the license already with your project that would be better if you don't mind.

Comment (marcelotrevisani) [reply recipes/skinnywms/meta.yaml]: However, for now you can let it as it is, but for the next versions would be good to have the license already shipped with your project
:)

Comment (sylvielamythepaut) [reply recipes/skinnywms/meta.yaml]: Can I put the text in the meta.yaml itself ?

**Comment by sylvielamythepaut** [comment]
Hi,

I have added the LICENSE.txt and updated the MANIFEST in the http://github.com/ecmwf/skinnywms

File: recipes/skinnywms/meta.yaml (hunk 6 above)
Comment (marcelotrevisani) [reply recipes/skinnywms/meta.yaml]: What do you mean?

File: LICENSE.txt [hunk 7]
```diff
@@ -1,27 +1,13 @@
...
-  and/or other materials provided with the distribution.
-
-* Neither the name of staged-recipes nor the names of its
-  contributors may be used to endorse or promote products derived from
-  this software without specific prior written permission.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
```
Comment (marcelotrevisani) [inline LICENSE.txt]: please, revert this change

Comment (marcelotrevisani) [reply LICENSE.txt]: This license is related to the ``conda-forge/staged-recipes`` and not your recipe

Comment (sylvielamythepaut) [reply LICENSE.txt]: Sorry, I was confused between the 2.

Comment (sylvielamythepaut) [reply LICENSE.txt]: Now I see .. I put the conda forge license back .

Comment (marcelotrevisani) [reply LICENSE.txt]: You reverted the wrong one  :(
Could you please revert this one ? and you can let your original license here
:)

**Comment by marcelotrevisani** [comment]
Hi @sylvielamythepaut , I made some modifications in your PR.
If you agree if those modifications I will merge it

**Comment by sylvielamythepaut** [comment]
Hi @marcelotrevisani , Thanks for helping me sorting out this license.
I noticed also that I had forgotten to add LICENSE.txt in https://github.com/ecmwf/skinnywms
I will add it now .
Then I am very happy with your modifications
Thanks again

**Comment by marcelotrevisani** [comment]
I am going to merge this PR then!
Thanks for your contribution! :tada:
